tests/baselines/reference/tsserver/typeReferenceDirectives/when-typeReferenceDirective-contains-UpperCasePackage.js

```diff
@@ -117,7 +166,6 @@ Info seq  [hh:mm:ss:mss] 	Files (4)
 	  Matched by default include pattern '**/*'
 	../lib/@types/UpperCasePackage/index.d.ts
 	  Entry point for implicit type library 'UpperCasePackage'
-	  Type library referenced via 'UpperCasePackage' from file '../lib/@app/lib/index.d.ts'
```

Comment:
This stops showing as this because of how this is processed. When the `UpperCasePackage` typings are already found and are primary then they are not processed again and hence the file reason for this is never added. Thats a separate issue and am looking into updating that code as well but its not related to this change. 

it just stopped showing because before this the package name by auto type ref was "UpperCasePackage" while by type Ref was "uppercasepackage" which made two distinct entries and hence processed again.